CRM/Financial/BAO/ExportFormat.php

```diff
@@ -105,14 +103,6 @@ public function getFileExtension() {
   }
 
   /**
-   * Override this if appropriate.
-   * @return null
-   */
-  public function getTemplateFileName() {
-    return NULL;
```

Comment:
I believe that the function isn't even being used in the entire Financial directory (grep call returns nothing). Also the current class has no parent. Should be pretty safe to delete this.